...\issue-04>python -m pytest
================================================= test session starts =================================================
platform win32 -- Python 3.8.5, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: ...\issue-04
plugins: anyio-3.3.1
collected 4 items

test_issue_04.py ..F.                                                                                            [100%]

====================================================== FAILURES =======================================================
_____________________________________________________ test_colors _____________________________________________________

    def test_colors():
        actual = ('blue', [0, 1])
        expected = fit_transform(['red', 'blue', 'blue'])
>       assert actual in expected
E       AssertionError: assert ('blue', [0, 1]) in [('red', [0, 1]), ('blue', [1, 0]), ('blue', [1, 0])]

test_issue_04.py:48: AssertionError
=============================================== short test summary info ===============================================
FAILED test_issue_04.py::test_colors - AssertionError: assert ('blue', [0, 1]) in [('red', [0, 1]), ('blue', [1, 0]),...
============================================= 1 failed, 3 passed in 0.25s =============================================